src/bindings/c/docs/how_to_write_unite_test.md

```diff
@@ -0,0 +1,33 @@
+# How to Write Unite Test for C API
+
+To ensure the accuracy of C API, all interfaces need to implement function level unite test at least. According to the object define, unite test cases are classified into following components:
+
+| Unite Case Component | Location | Description |
+|:---     |:---   |:---
+|Core|[ov_core_test.cpp](../tests/ov_core_test.cpp)| including all core related interfaces tests
+|Model|[ov_model_test.cpp](../tests/ov_model_test.cpp)| including all model related interfaces tests
+|Compiled Model|[ov_compiled_model_test.cpp](../tests/ov_compiled_model_test.cpp)| including all compiled model related interfaces tests
+|Infer Request|[ov_infer_request_test.cpp](../tests/ov_infer_request_test.cpp)| including all infer request related interfaces tests
+|Tensor|[ov_tensor_test.cpp](../tests/ov_tensor_test.cpp)| including all tensor related interfaces tests
+|Partial Shape|[ov_partial_shape_test.cpp](../tests/ov_partial_shape_test.cpp)| including all partial shape related interfaces tests
+|Layout|[ov_layout_test.cpp](../tests/ov_layout_test.cpp)| including all layout related interfaces tests
+|Preprocess|[ov_preprocess_test.cpp](../tests/ov_preprocess_test.cpp)| including all preprocess related interfaces tests
```

Comment:
I think we have a good file names which describe purpose for each file. I am not sure that we need to have this table. Just we can mention that all tests are located in the folder.

Reply:
removed